app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewActivity.kt

```diff
@@ -60,11 +68,28 @@ class PledgedProjectsOverviewActivity : AppCompatActivity() {
                         modifier = Modifier,
                         onBackPressed = { onBackPressedDispatcher.onBackPressed() },
                         lazyColumnListState = lazyListState,
+                        errorSnackBarHostState = snackbarHostState,
                         ppoCards = ppoCardPagingSource,
-                        totalAlerts = totalAlerts.value
+                        totalAlerts = totalAlerts.value,
+                        onSendMessageClick = { projectName -> viewModel.onMessageCreatorClicked(projectName) }
                     )
                 }
 
+                LaunchedEffect(Unit) {
+                    viewModel.projectFlow
+                        .collect {
+                            startCreatorMessageActivity(it, previousScreen = MessagePreviousScreenType.PLEDGED_PROJECTS_OVERVIEW)
+                        }
+                }
+
+                val coroutineScope = rememberCoroutineScope()
+
+                viewModel.provideSnackbarMessage {
+                    coroutineScope.launch {
```

Comment:
Not sure if this is the best way to do this, but `showSnackBar` must be called from a coroutine or other suspend function.

Reply:
You can see my comment for a recommendation/link for more info on what to use here